website/pages/docs/layout/container.mdx

```diff
@@ -40,6 +40,19 @@ To contain any piece of content, wrap it in the `Container` component.
 </Container>
 ```
 
+## Container Size
+
+To set the size of a Container, use the `maxW` attribute.
+
+```jsx
+<VStack>
+  <Container maxW="container.xl"> Extra-Large Container </Container>
+  <Container maxW="container.lg"> Large Container </Container>
+  <Container maxW="container.md"> Medium Container </Container>
+  <Container maxW="container.sm"> Small Container </Container>
```

Comment:
Tim was probably refering to the spaces after opening `>` and before closing `<`.
I thought it is incorrect formatting too, but apparently prettier does not care about that. If you remove them, they won't be readded. 🤷‍♂️
But it's a inconsistency with the rest of our docs/codebase, so it'd be better to remove them.
Thanks.